react/StyleGuideProvider/StyleGuideProvider.js

```diff
@@ -7,6 +7,10 @@ import classnames from 'classnames';
 import Helmet from 'react-helmet';
 import ScreenReaderOnly from '../ScreenReaderOnly/ScreenReaderOnly';
 
+if (typeof window !== 'undefined') {
+  require('smoothscroll-polyfill').polyfill();
```

Comment:
From an experience point of view it would be nice to be consistent on this. While this particular use case is to support the `AutoSuggest` component, by polyfilling at this level any consuming single page app that used `document.body.scrollIntoView()` to scroll to top could have the uniform smoothness. Thoughts @mattsjones @markdalgleish?